client/extensions/woocommerce/rest-api-client/actions.js

```diff
@@ -0,0 +1,41 @@
+
+/**
+ * Internal dependencies
+ */
+import {
+	WOOCOMMERCE_API_CLIENT_ERROR,
+	WOOCOMMERCE_API_CLIENT_RECEIVED,
+	WOOCOMMERCE_API_CLIENT_REQUESTED,
+} from './action-types';
+
+export function requestedAction( siteKey, endpoint, ids ) {
+	return {
+		type: WOOCOMMERCE_API_CLIENT_REQUESTED,
+		apiType: 'woocommerce',
```

Comment:
Should this be hardcoded here? Are you making this open to any API endpoint namespace, or is this just for the WC endpoints? (following that, should the actions use a different prefix?)

Reply:
Oops, you're totally right. `apiType` should be an argument.

Reply:
And also, the actions should totally use a different prefix. I'm only prefixing them with `WOOCOMERCE_` for now while they exist under `client/extensions/woocommerce`.